lib/portage/repository/config.py

```diff
@@ -340,7 +351,7 @@ def __init__(self, name, repo_opts, local_config=True):
 
 			for value in ('allow-missing-manifest',
 				'cache-formats',
-				'create-manifest', 'disable-manifest', 'manifest-hashes',
+				'create-manifest', 'disable-manifest', 'manifest-hashes', 'maintainer-nag', 'maintainer-nag-url',
 				'manifest-required-hashes', 'profile-formats', 'properties-allowed', 'restrict-allowed',
 				'sign-commit', 'sign-manifest', 'thin-manifest', 'update-changelog'):
 				setattr(self, value.lower().replace("-", "_"), layout_data[value])
```

Comment:
The layout.conf attributes overwrites the repos.conf settings here. This is intentional in the sense that we don't want to allow repos.conf to override some settings, so maybe add a second layout.conf loop for settings that repos.conf is allowd to override.